src/crypto/client/NUT11.ts

```diff
@@ -262,19 +262,21 @@ export const signP2PKProofs = (
  * requires a signature from the key.
  *
  * @param proof - A proof to sign.
- * @param privateKey - A single private key.
+ * @param privateKey - A single private key (hex string or Uint8Array).
  * @throws Error if signature is not required or proof is already signed.
  */
-export const signP2PKProof = (proof: Proof, privateKey: string): Proof => {
+export const signP2PKProof = (proof: Proof, privateKey: string | Uint8Array): Proof => {
```

Comment:
Widening privateKey to string | Uint8Array will inadvertently create an undocumented breaking change for v3, but it's not a bad idea, so we should add it to https://github.com/cashubtc/cashu-ts/pull/432/

Reply:
Have raised in the other PR, so marking this as resolved.